ansible/environments/local/group_vars/all

```diff
@@ -17,9 +17,13 @@ db_port: "{{ lookup('ini', 'db_port section=db_creds file={{ playbook_dir }}/db_
 limits:
   actions:
     invokes:
-      perMinute: 60
+      blockingTimeout: 60
       concurrent: 30
       concurrentInSystem: 5000
+      perMinute: 60
+  activations:
+    polls:
+      maxRecords: 200
```

Comment:
i thought there was a limit on all list operations; it looks like for activations it is 200 but for the others, only a default is provided and no limit... this is an oversight.